packages/lib-user/src/components/MyGroups/MyGroups.js

```diff
@@ -13,20 +16,69 @@ const StyledGrid = styled(Grid)`
 `
 
 function MyGroups({
-  children
+  error = undefined,
+  groups = [],
+  loading = false
 }) {
   return (
-    <StyledGrid
-      forwardedAs='ul'
-      pad='none'
-    >
-      {children}
-    </StyledGrid>
+    <>
+      {loading ? (
+        <Box
+          align='center'
+          fill
+          justify='center'
+          pad='medium'
+        >
+          <Loader />
+        </Box>
+      ) : error ? (
+        <Box
+          align='center'
+          fill
+          justify='center'
+          pad='medium'
+        >
+          <SpacedText uppercase={false}>
+            There was an error.
+          </SpacedText>
+          <SpacedText uppercase={false}>
+            {error?.message}
+          </SpacedText>
+        </Box>
+      ) : !groups?.length ? (
+        <Box
+          align='center'
+          fill
+          justify='center'
+          pad='medium'
+        >
+          <Paragraph margin={{ top: '0', bottom: '20px' }}>
+            You are not a member of any Groups.
+          </Paragraph>
+          <Paragraph margin={{ top: '0', bottom: '20px' }}>
+            Create one below
+          </Paragraph>
```

Comment:
For translations sake, I suggest consolidating this message and others like it into one Paragraph. You can limit the width of the containing Box to bump certain words to the next line, or include a `\n` because it can be handled by `i18n`.

When phrases are split like this between multiple Paragraph components, there must be a translation key for each, and context gets lost when a pro is asked to translate one of the phrases in isolation.

Reply:
I can't seem to get `\n` to work within <Paragraph> or <SpacedText>, is there an example I could reference?

Reply:
Definitely need to refactor these (and other similar) content for translations, but I think I'll merge this for now and will revisit. I'm probably missing something obvious, but want to get these changes in for continued testing.

Reply:
The translation strings used in lib-classifier's Banner components is an example of multi-line text contained in one key. WorkflowIsFinishedBanner in Storybook: [link](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/banners-workflowisfinishedbanner--default).

In the translation dictionary:
https://github.com/zooniverse/front-end-monorepo/blob/8519583d2d2d1f6a703039e0456a53e97c6c110a/packages/lib-classifier/src/translations/en/components.json#L45-L47